bus-mapping/src/evm/opcodes/return.rs

```diff
@@ -42,6 +43,27 @@ impl Opcode for Return {
                 let code = memory.0[offset..offset + length].to_vec();
                 state.code_db.insert(code);
             }
+
+            state.call_context_write(
+                &mut exec_step,
+                state.caller()?.call_id,
+                CallContextField::LastCalleeId,
+                state.call()?.call_id.into(),
+            );
+
+            state.call_context_write(
+                &mut exec_step,
+                state.caller()?.call_id,
+                CallContextField::LastCalleeReturnDataOffset,
+                offset.into(),
+            );
+
+            state.call_context_write(
+                &mut exec_step,
+                state.caller()?.call_id,
+                CallContextField::LastCalleeReturnDataLength,
+                length.into(),
+            );
```

Comment:
After rebase on `main`, these should be handled in `handle_restore_context` and could be removed.

Reply:
Is this PR: https://github.com/xiaodino/zkevm-circuits/commit/809895b75630c6ee42eadfba85e8274389a20c5b the one that handle `state.call_context_write`?

Reply:
Updated. Please take a look